acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/EthSignerProcessRunner.java

```diff
@@ -196,9 +198,9 @@ public void start(final String processName) {
     if (signerConfig.serverTlsOptions().isPresent()) {
       final TlsOptions serverTlsOptions = signerConfig.serverTlsOptions().get();
       params.add("--tls-keystore-file");
-      params.add(serverTlsOptions.getKeyStoreFile().toString());
+      params.add(serverTlsOptions.getStoreFile().toString());
       params.add("--tls-keystore-password-file");
-      params.add(serverTlsOptions.getKeyStorePasswordFile().toString());
+      params.add(serverTlsOptions.getStorePasswordFile().toString());
```

Comment:
'getStore' is pretty difficult to read, it might have been better left as 'getKeyStore'...

Reply:
This has been reverted.